packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -311,7 +311,7 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
               onMouseDown={this._onDropdownMouseDown}
               onFocus={this._onFocus}
             >
-              <span id={this._optionId} className={this._classNames.title} aria-live="polite" aria-atomic={true}>
+              <span id={this._optionId} className={this._classNames.title} aria-live={hasFocus ? 'polite' : undefined}>
```

Comment:
I think this should be

```suggestion
              <span id={this._optionId} className={this._classNames.title} aria-live={hasFocus ? 'polite' : undefined} aria-atomic={hasFocus ? true : undefined}>
```

Reply:
oh yeah..totally missed that in the cherry pick